sdk/tests/AccountCreateTransactionTest.cc

```diff
@@ -48,12 +49,14 @@ TEST_F(AccountCreateTransactionTest, ConstructAccountCreateTransaction)
   EXPECT_EQ(transaction.getAutoRenewPeriod(), std::chrono::hours(2160)); // 90 days
   EXPECT_EQ(transaction.getTransactionMemo(), std::string());
   EXPECT_EQ(transaction.getMaxAutomaticTokenAssociations(), 0U);
-  EXPECT_FALSE(transaction.getStakedAccountId());
-  EXPECT_FALSE(transaction.getStakedNodeId());
+  EXPECT_FALSE(transaction.getStakedAccountId().has_value());
+  EXPECT_FALSE(transaction.getStakedNodeId().has_value());
   EXPECT_FALSE(transaction.getDeclineStakingReward());
   EXPECT_EQ(transaction.getAlias(), nullptr);
+  EXPECT_FALSE(transaction.getEvmAddress().has_value());
 }
 
+//-----
```

Comment:
Do we need this empty comment line here?

Reply:
It's not, though I like to add them to source files to signify the start of a function. Helps with organization a bit for me but it can be removed if it bothers you